runhouse/main.py

```diff
@@ -798,9 +806,12 @@ def restart(
         False,
         help="Whether HTTP server started from inside a Python call rather than CLI.",
     ),
+    log_level: str = typer.Option(
+        default=DEFAULT_LOG_LEVEL, help="The lowest log level of the printed logs"
+    ),
 ):
     """Restart the HTTP server on the cluster."""
-    if name:
+    if name and log_level == DEFAULT_LOG_LEVEL:
```

Comment:
not sure I follow this logic - why do we treat restart differently if the log level is "INFO"?